bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/vehicle-update-channel-types.xml

```diff
@@ -6,6 +6,10 @@
 	<channel-type id="state-update-channel">
 		<item-type>Switch</item-type>
 		<label>trigger the update of the state</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
Question: do you need a property here? This channel has no state, it just sends an update command (autoUpdatePolicy = veto).

Reply:
Removed